testdata/job.yaml

```diff
@@ -0,0 +1,36 @@
+engine: 2
+verifier: 2
+job_spec_docker:
+  image: gromacs/gromacs
+  entrypoint:
+    - /bin/bash
+    - -c
+    - echo 15 | gmx pdb2gmx -f input/1AKI.pdb -o output/1AKI_processed.gro -water spc
+  env: []
+job_spec_language:
```

Comment:
make it so you only have to specify one of `docker` or `language`, please

you can do this with `omitempty` in the `yaml` and `json` struct annotation fields